src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/GAOrderMadeTaskHandler.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.civil.handler.tasks;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.camunda.bpm.client.task.ExternalTask;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+import uk.gov.hmcts.reform.civil.service.search.OrderMadeSearchService;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.time.LocalDate.now;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.CHECK_ORDER_MADE_END_DATE;
+import static uk.gov.hmcts.reform.civil.enums.dq.GeneralApplicationTypes.STAY_THE_CLAIM;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+@ConditionalOnExpression("${response.deadline.check.event.emitter.enabled:true}")
```

Comment:
shouldnt this be different to response ?

Reply:
Modified the property name to judge-revisit-event-emitter